mcs/class/corlib/corlib.dll.sources

```diff
@@ -1661,3 +1661,7 @@ corefx/SR.cs
 ../../../external/corefx/src/System.Collections.Concurrent/src/System/Collections/Concurrent/OrderablePartitioner.cs
 ../../../external/corefx/src/System.Collections.Concurrent/src/System/Collections/Concurrent/Partitioner.cs
 ../../../external/corefx/src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs
+
+System.Security.Cryptography.X509Certificates/X509CertificateImplApple.cs
```

Comment:
No, we're already doing this in monotouch_corlib.dll.sources and the non-mobile IntPtr version is here as well (refactoring this is outside of scope).